core/modules/layout/css/layout.admin.css

```diff
@@ -406,3 +411,6 @@
 .layout-editor-block-disabled {
   color: #999;
 }
+.form-item.form-item-field-blocks {
```

Comment:
We should add another selector here to make sure we're within the layout add form.

Reply:
Actually looks like we should just use the Form API `#indentation` attribute to indent this field.